garak/detectors/riskywords.py

```diff
@@ -7,11 +7,12 @@
 import csv
 
 from garak import _config
+from garak.data import path as data_path
 from garak.detectors.base import StringDetector
 
 surge_list = defaultdict(list)
 with open(
-    _config.transient.package_dir / "resources" / "profanity_en.csv",
+    data_path / "profanity_en.csv",
```

Comment:
Probably worth looking at as part of a larger refactor -- do we want all these datasets in our git? Or should we have them in some other place _e.g._ HuggingFace hub, and the `garak.data` module can also manage downloading these files?

Reply:
Have discussed in the past that we may want tooling that will download all datasets to create an `offline` deployment capability.

I suspect there could be some expansion on `garak.data` for handling access to known datasets. Treating HF as the specific location that tooling can register data as available from might be a good direction.